src/Telegram.Bot/Exceptions/DefaultExceptionParser.cs

```diff
@@ -93,5 +97,33 @@ private static string TryTruncateErrorDescription(string message, string descrip
                 message = message.Substring(description.Length);
             return message;
         }
+
+        /// <inheritdoc cref="IExceptionParser"/>
+        public async Task<Exception> ParseResponseAsync(HttpResponseMessage httpResponse)
+        {
+            Exception exception;
+
+            string responseJson = await httpResponse.Content.ReadAsStringAsync()
+                .ConfigureAwait(false);
+
+            var apiResponse = JsonConvert.DeserializeObject<JObject>(responseJson);
+            bool? ok = apiResponse["ok"].Value<bool?>();
+            string description = apiResponse["description"].Value<string>();
+
+            if (ok == true)
+            {
+                exception = new ApiRequestException(description);
+            }
+            else if (ok == false)
+            {
+                exception = new ApiRequestException(description);
+            }
+            else
+            {
+                exception = new ApiRequestException("Invalid status code: " + httpResponse.StatusCode);
+            }
+
+            return exception;
```

Comment:
Do I understand correctly, base parser will only return ApiRequestException? And it seems that `HttpStatusCode` was lost on the way :(